test/core/util/test_config.cc

```diff
@@ -66,9 +66,9 @@ int64_t grpc_test_sanitizer_slowdown_factor() {
   } else if (BuiltUnderAsan()) {
     sanitizer_multiplier = 3;
   } else if (BuiltUnderMsan()) {
-    sanitizer_multiplier = 4;
+    sanitizer_multiplier = 15;
```

Comment:
Increasing this multiplier seems like a good idea!  However, if we make it too high, we might make test runs take a lot longer, so how about choosing a smaller value and then increasing it again if it's not big enough?  I'm thinking something like 8 for msan and 10 for ubsan -- doubling the current value in both cases.

Reply:
I'm not suggesting to manually kick off an RBE run with different values.  I'm suggesting changing this PR to use 8 and 10 and merging it, and seeing what that does to the flake rate.  If it doesn't decrease it enough, we can change the values again in a subsequent PR.